config/packages/doctrine.yaml

```diff
@@ -4,7 +4,7 @@ doctrine:
 
         # IMPORTANT: You MUST configure your server version,
         # either here or in the DATABASE_URL env var (see .env file)
-        #server_version: '5.7'
+        server_version: '5.7'
```

Comment:
You probably should specify your server version in the database_url in your .env.local because the 5.7 mysql server version is probably not the version for everyone

Reply:
Ok, i removed that, and i removed some migrations which were not useful.